src/services/member/controller.test.ts

```diff
@@ -185,4 +214,225 @@ describe('Member Controller', () => {
       expect(mockSendEmail).not.toHaveBeenCalled();
     });
   });
+  describe('GET /members/current/storage/files', () => {
+    it('returns ok', async () => {
+      const response = await app.inject({
+        method: HttpMethod.Get,
+        url: '/members/current/storage/files',
+      });
+      expect(response.statusCode).toBe(StatusCodes.OK);
+    });
+    it('returns bad request when page is lower than 1', async () => {
+      const response = await app.inject({
+        method: HttpMethod.Get,
+        url: '/members/current/storage/files',
+        query: { page: FILE_METADATA_MIN_PAGE - 1 + '' },
+      });
+      expect(response.statusCode).toBe(StatusCodes.BAD_REQUEST);
+    });
+    it('returns bad request when page size is lower than 1', async () => {
+      const response = await app.inject({
+        method: HttpMethod.Get,
+        url: '/members/current/storage/files',
+        query: { pageSize: FILE_METADATA_MIN_PAGE_SIZE - 1 + '' },
+      });
+      expect(response.statusCode).toBe(StatusCodes.BAD_REQUEST);
+    });
+    it('returns bad request when page size is greater than 50', async () => {
```

Comment:
Not sure if it is working:

```suggestion
    it(`returns bad request when page size is greater than ${FILE_METADATA_MAX_PAGE_SIZE}`, async () => {
```

Reply:
Having variables in the test name makes it difficult to be runned independently from command line or Jest Runner.
What about `returns bad request when page size is greater than the maximum` ?

Reply:
Okay I understand. Yeah I think it's good :). It was okay with the `50`, my only issue is if we change the limit we have to remember to change the test name too. Not a big deal, so it's as you want.

Reply:
Yeah, I agree with you about the number!